sanic/app.py

```diff
@@ -352,13 +352,13 @@ def middleware(self, middleware_or_request):
                            attach_to=middleware_or_request)
 
     # Static Files
-    def static(self, uri, file_or_directory, pattern=r'/?.+',
+    def static(self, uri, file_or_directory, host=None, pattern=r'/?.+',
```

Comment:
when adding a parameter it's always better to add at the end rather than in the middle